test/travis.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
this works, but i expect this script will grow, and using bash rather than POSIX sh is prob easier for people to work under.  so i'd suggest using /bin/bash now.